__tests__/__system__/cli/delete/uss-file/__snapshots__/cli.delete.uss-file.test.ts.snap

```diff
@@ -32,6 +32,13 @@ exports[`delete uss file command should display delete uss file help 1`] = `
 
       Specify this option to confirm that you want to delete the data set permanently.
 
+ OPTIONS
+ -------
+
+   --recursive  (boolean)
+
+      Delete the directory and all files/directories under it.
+
    --host  | -H (string)
```

Comment:
Not sure on the UX of this, but what if we sorted all the options alphabetically?

Reply:
Hi @zFernand0 , thank you for the question. Usually the global options are after the command specified options and the order is based on user habits. I think ours are ok now. 😊